### PR TITLE
[ja] add Japanese translation of "install the collector"

### PR DESCRIPTION
**Pull request by ymotongpoo**
Translate this page into Japanese: https://opentelemetry.io/docs/collector/installation/

### CODE REVIEW

**Comment by theletterf** [comment]
@open-telemetry/docs-ja-approvers Please have a look.

File: content/ja/docs/collector/installation.md [hunk 1]
```diff
@@ -0,0 +1,296 @@
+---
+title: コレクターのインストール
```
Comment (katzchang) [inline content/ja/docs/collector/installation.md:2]: 原文が "Install the Collector" で、大文字始まりのCollectorなので、英語表記のまま「Collectorのインストール」にしたい。
以下の文章中も、大文字始まりCollectorの箇所は英語表記のままにしたい。

File: content/ja/docs/collector/installation.md [hunk 2]
```diff
@@ -0,0 +1,296 @@
...
+    - 8889:8889 # Prometheus exporter metrics
+    - 13133:13133 # health_check extension
+    - 4317:4317 # OTLP gRPC receiver
+    - 4318:4318 # OTLP http receiver
+    - 55679:55679 # zpages extension
+```
+
+## Kubernetes
+
+次のコマンドは、エージェントをデーモンセットと単一のゲートウェイインスタンスとしてデプロイします。
```
Comment (katzchang) [inline content/ja/docs/collector/installation.md:72]: 「デーモンセットとしてエージェントと、単一のゲートウェイインスタンスをデプロイします」

**Review by katzchang — CHANGES_REQUESTED** [review]

**Comment by theletterf** [comment]
@ymotongpoo Could you check the suggestions? Thanks!

**Comment by svrnm** [comment]
@ymotongpoo any updates?

**Comment by ymotongpoo** [comment]
It turned out that I need to recreate the PR with new company email address so that EasyCLA process properly understand the requester.

**Comment by svrnm** [comment]
> @svrnm @theletterf sorry for the delay. I've been in the process of changing jobs and have had no time for this. I'm trying to make time.

No worries, we check in from time to time with PRs pending, but we all understand that live changes and priorties shift. Glad to hear that you found a new place to work for and still want to find time to work on this.

> It turned out that I need to recreate the PR with new company email address so that EasyCLA process properly understand the requester.

Yes, that might be a thing, if you need to close this PR and raise a new one don't hesitate to do that

**Comment by ymotongpoo** [comment]
Thank you, @svrnm for checking in and suggestion! Closing this PR for now and I'll reopen with new email address marked.

**Pull request closed without merging**